lib/tasks/canonical_models/alchemical_properties.json

```diff
@@ -207,6 +207,14 @@
       "effects_cumulative": false
     }
   },
+  {
+    "attributes": {
+      "name": "Fortify Speech",
```

Comment:
This property doesn't exist on any ingredients (except through [Creations](https://en.uesp.net/wiki/Skyrim:Creation_Club)) but there are potions in-game with these effects.